draft-ietf-rats-architecture.md

```diff
@@ -374,41 +374,21 @@ An Attester creates Evidence that is conveyed to a Verifier.
 A Verifier uses the Evidence, any Reference Values from Reference Value Providers, and any Endorsements from Endorsers,
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester.
 This procedure is called the appraisal of Evidence.
-
 Subsequently, the Verifier generates Attestation Results for use by Relying Parties.
-The Appraisal Policy for Evidence
-might be obtained from an Endorser along with the Endorsements, and/or might be obtained
-via some other mechanism, such as being configured in the Verifier by the Verifier Owner.
+
+The Appraisal Policy for Evidence might be obtained from the Verifier Owner,  
+and/or is configured into the Verifier by the Verifier Owner, 
```

Comment:
These two are the same thing.  Also this change doesn't appear to be related to issue #325